inception-project/inception
# #1336 - Add dark mode

**What's in the PR**
* Added experimental dark mode

**How to test manually**
* Add `ui.dark-mode.enabled=true` to the `settings.properties` file
* Now you should have a sun/moon icon in the menu bar that can be used to switch between the modes
* Check where dark mode looks bad
* Mind that annotation editors won't respect the light/dark mode yet - but the general page frames, settings and most non-editor parts of the UI should consider the mode

**Automatic testing**

**Documentation**
